### ISSUE TITLE
Conflicting E241 and E261

### ISSUE THREAD

**Issue by pajaco** [PyCQA/pycodestyle#639]
**Version:** 2.3.1

**Problem:**
Inline comments which are prepended with at least 2 spaces (as enforced by E261) are reported as violations of E241.

**Expected:**
The E241 rule should only be enforced when the next character after `<comma><space><space>` sequence is not `'#'`.

*Example*
```python
[
    1,  # Inline comment
    2,
]
```

**Comment by sigmavirus24** [comment]
That is exactly why all `E24` errors are [ignored by default][]. You must be enabling.

[ignored by default]: https://github.com/PyCQA/pycodestyle/blob/f1908ba99b72c272720de067794cd3d1a88a1bbb/pycodestyle.py#L72

**Comment by asottile** [comment]
cannot reproduce on 2.5.0
